src/odfesqlodbc/es_result_queue.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright <2019> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+#define QUEUE_CAPACITY 2
+#include "es_result_queue.h"
+
+ESResultQueue::ESResultQueue()
+#ifdef __APPLE__
```

Comment:
You can get rid of this warning, since there's only one parameter here.
(verified with a local Mac build)